### ISSUE TITLE
The performance on shapenet part segmentation

### ISSUE THREAD

**Issue by liuxinhai** [yangyanli/PointCNN#174]
Coud you offer pretrained model for shapenet part segmentation?
I have tried to trian a new model, but the mIoU of shapenet part segmentation is only 85.23% which is much lower than 86.14% in the paper.
Thanks!

**Comment by burui11087** [comment]
Hi @liuxinhai 

Sorry for replying you so late. We train shapenet part segmentation network on P100 for about 5 days and select the best accuracy model show in tensorboard. It's a pity that we can't find our model but I think it's easy to reproduce the result.

Thanks
